jibri/rootfs/etc/cont-init.d/11-get-https-cert

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/with-contenv bash
+
+while [ ! -f /usr/local/share/ca-certificates/meet.crt ]
+do
+  sleep 1
+  if CERTS=$(openssl s_client -showcerts -connect meet.jitsi:443 < /dev/null); then
```

Comment:
the openssl works for me only with SNI like this
`openssl s_client -showcerts -connect meet.jitsi:443 -servername meet.jitsi`